cpp/unreal_plugins/UrdfBot/Source/UrdfParser.cpp

```diff
@@ -0,0 +1,381 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "UrdfParser.h"
+
+#include <XmlFile.h>
+#include <XmlNode.h>
+#include <XmlParser.h>
+
+#include "Assert/Assert.h"
+
+UrdfRobotDesc UrdfParser::parse(const std::string& file_name)
+{
+    FXmlFile file;
+    file.LoadFile(UTF8_TO_TCHAR(file_name.c_str()));
+
+    FXmlNode* robot_node = file.GetRootNode();
+    ASSERT(robot_node->GetTag().Equals(TEXT("robot")));
+
+    return parseRobotNode(robot_node);
+}
+
+UrdfRobotDesc UrdfParser::parseRobotNode(FXmlNode* robot_node)
+{
+    UrdfRobotDesc robot_desc;
+    // parse all nodes
```

Comment:
Insert new line whitespace between lines 26 and 27.

Reply:
fixed